webapp/src/views/projects/members/ProjectMembersView.tsx

```diff
@@ -72,11 +69,11 @@ export const ProjectMembersView: FunctionComponent = () => {
       {project.organizationOwnerSlug && (
         <Box mb={2}>
           <Typography component={Box} alignItems={'center'} variant={'body1'}>
-            <T>project_permission_information_text_base_permission_before</T>{' '}
-            {basePermissionText}
+            <T keyName="project_permission_information_text_base_permission_before" />{' '}
+            {translatePermission(project.organizationOwnerBasePermissions!)}
```

Comment:
Wouldn't this be problematic for languages where these would have to be in a different order?